com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/service/DocumentExtensionTests.java

```diff
@@ -82,11 +82,11 @@ private void checkExtensionMatches(String extension) {
     fireDidOpen(extension, uri);
     verify(engine, timeout(10000).times(1)).analyze(uri, TEXT, CopybookProcessingMode.ENABLED);
 
-    when(engine.analyze(uri, TEXT, CopybookProcessingMode.DISABLED))
+    when(engine.analyze(uri, TEXT, CopybookProcessingMode.ENABLED))
         .thenReturn(AnalysisResult.empty());
     fireDidChange(uri);
     verify(engine, timeout(10000))
-        .analyze(uri, INCORRECT_TEXT_EXAMPLE, CopybookProcessingMode.DISABLED);
+        .analyze(uri, INCORRECT_TEXT_EXAMPLE, CopybookProcessingMode.SKIP);
```

Comment:
Shouldn't the modes here be the same?

Reply:
got it - ok, I was focused on the verify, fixed